app/views/jobs/show.html.erb

```diff
@@ -14,5 +14,5 @@ Company: <%= link_to @job.company.name, company_path(@job.company) %><br/>
 
 Tags:
 <% @tags.each do |tag| %>
-  <p><%= tag.title %> - <%= tag.jobs.count %> ("<%= Tag.average_salary(tag) %>")</p>
+  <p><%= tag.title %> - <%= tag.jobs.count %> (<%= Tag.average_salary(tag) %>)</p>
```

Comment:
What led to the decision for making this a class method that takes an instance of a tag as an argument vs just having an instance method called on that instance of tag?